pagopa/pn-frontend
# fix pull request base branch for pr

## Short description
Fixed some problems with branch detection in sonar pipeline.

## How to test
Go to sonarCloud and check that, for each pull request analysis or branch analysis the target branch is correct.
Check also that for pn-personafisica-login coverage esclusions are finally fixed.
